src/aparnaVDec19/assignment2/FindMissingNumberInArray.java

```diff
@@ -0,0 +1,27 @@
+package aparnaVDec19.assignment2;
+
+/*
+ * 1) Find the missing number in integer array of 1 to 10?
+      Hint: You have given an integer array which contains numbers from 1 to 10 
+      but one number is missing, you need to write a 
+      Java program to find that missing number in an array.
+ */
+public class FindMissingNumberInArray {
+
+	static void missingNumber(int[] arr) {
+		int number = 10;
```

Comment:
here you passing number 10 which is highest, but you should do first step is like find max number from array and that max number should be your number.
this is also correct but i giving you dynamic idea